drivers/mcp2515/mcp2515.c

```diff
@@ -0,0 +1,412 @@
+/*
+ * Copyright (C) 2016 OTA keys
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_mcp2515
+ * @{
+ *
+ * @file
+ * @brief       mcp2515 can spi driver
+ *
+ * @author      Toon Stegen <toon.stegen@altran.com>
```

Comment:
Right authorship?

Reply:
See explanation above. (Marking this one as resolved, will keep other one open for discussion)